src/styles/index.scss

```diff
@@ -1,3 +1,222 @@
-body {
+.page {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+p {
+  margin: 0;
+}
+
+.header {
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  display: flex;
+  box-shadow: 2px 4px #0000000d;
+
+  &__logo {
+    align-items: center;
+    width: 40px;
+    height: 40px;
+    padding: 0 50px;
+    display: flex;
+  }
+}
+
+.nav {
+  margin: 0 50px;
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    font-size: 12px;
+    list-style: none;
+    display: flex;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    color: #060b35;
+    text-transform: uppercase;
+    padding-top: 22px;
+    padding-bottom: 22px;
+    font-family: Roboto, sans-serif;
+    font-weight: 500;
+    line-height: 16px;
+    text-decoration: none;
+    display: inline-block;
+    position: relative;
+  }
+
+  &__link--is-active {
+    color: #00acdc;
+    position: relative;
+  }
+
+  &__link--is-active::after {
+    content: ' ';
+    position: absolute;
+    bottom: 0;
+    display: block;
+    border-radius: 2px;
+    width: 100%;
+    height: 4px;
+    left: 0;
+    background: #00acdc;
+  }
+
+  &__link:hover {
+    color: #00acdc;
+  }
+}
+
+main {
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+  display: grid;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The `transform: scale(1.2);` on hover can cause layout shifts, which might not be desirable. Consider using a more subtle effect or ensuring that the surrounding layout can accommodate the scaling without shifting.